### PR TITLE
December news

### PR DESCRIPTION
**Pull request by maelle**
TODOs 

- [x] rebase

### CODE REVIEW

File: content/blog/2023-12-22-news-december-2023/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,645 @@
...
+
+This Giving Season consider donating to rOpenSci to support our mission of empowering Open Science.
+
+Sustaining an open project with quality infrastructure freely accessible to the global and diverse community of R software users, research software developers, and engineers requires many different resources. Our organization’s ongoing costs are supported by grants and donations from individuals and organizations which share our vision and mission.
+
+By supporting us, you’re not just donating; you’re contributing to a community that’s breaking down barriers, creating opportunities, and shaping the future of open and reproducible science for everyone. 
+
+Join us in this vital mission today! You can donate here: https://ropensci.org/donate/
+
+Thanks to those who already donated!
```
Comment (yabellini) [inline content/blog/2023-12-22-news-december-2023/index.Rmd:56]:
```suggestion
We are gratful for the donation we already receive. 
```

File: content/blog/2023-12-22-news-december-2023/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,645 @@
...
+
+rOpenSci Mark Padgham was interviewed on the R Consortium blog about his [deposits package](https://www.r-consortium.org/blog/2023/11/30/deposits-r-package-delivers-a-common-workflow-for-r-users) that simplifies data sharing.
+
+> "Publicly depositing datasets associated with published research is becoming more common, partly due to journals increasingly requiring data sharing and partly through more general and ongoing cultural changes with data sharing. Yet data sharing is often seen as time-consuming, particularly to meet the expectations of individual data repositories. While documentation and training can help familiarize users with data-sharing processes, browser-based data, and metadata submission workflows can only be so fast, are not easily reproduced, and do not facilitate regular or automated data and metadata updates. Better programmatic tools can transform data sharing from a mountainous climb into a pit of success."
+
+This interview was furthermore featured on the [R Weekly highlights podcast](https://rweekly.fireside.fm/145?t=848) hosted by Eric Nantz and Mike Thomas.
+
+### "A package a day" series on Mastodon!
```
Comment (steffilazerte) [inline content/blog/2023-12-22-news-december-2023/index.Rmd:66]: And weekly package digests on LinkedIn?

File: content/blog/2023-12-22-news-december-2023/index.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
...a given location and time according to the World Magnetic Model (WMM). Both the main field and secular variation components are returned. This functionality is useful for physicists and geophysicists who need orthogonal components from WMM. Currently, this package supports annualized time inputs between 2000 and 2025. If desired, users can specify which WMM version to use, e.g., the original WMM2015 release or the recent out-of-cycle WMM2015 release. Methods used to implement WMM, including the Gauss coefficients for each release, are described in the following publications: Chulliat et al (2020) <doi:10.25923/ytk1-yx35>, Chulliat et al (2019) <doi:10.25921/xhr3-0t19>, Chulliat et al (2015) <doi:10.7289/V5TB14V7>, Maus et al (2010) <https://www.ngdc.noaa.gov/geomag/WMM/data/WMMReports/WMM2010_Report.pdf>, McLean et al (2004) <https://www.ngdc.noaa.gov/geomag/WMM/data/WMMReports/TRWMM_2005.pdf>, and Macmillian et al (2000) <https://www.ngdc.noaa.gov/geomag/WMM/data/WMMReports/wmm2000.pdf>.  It is available on [CRAN]( https://CRAN.R-project.org/package=wmm). It has been [reviewed](https://github.com/ropensci/software-review/issues/522) by Athanasia Monika Mowinckel and Rohit Goswami.
```
Comment (steffilazerte) [inline content/blog/2023-12-22-news-december-2023/index.md:91]: Perhaps trim this description down a bit?

File: content/blog/2023-12-22-news-december-2023/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,645 @@
...
+See also our [help wanted page](/help-wanted/) -- before opening a PR, we recommend asking in the issue whether help is still needed.
+
+## Package development corner
+
+Some useful tips for R package developers. :eyes:
+
+### An older post about dependencies
+
+Do you know of the post ["Playing on the same team as your dependency"](https://www.tidyverse.org/blog/2022/09/playing-on-the-same-team-as-your-dependecy/) by Thomas Lin Pedersen?
+It's a take from the maintainer perspective, on how maintainers of reverse dependencies can help make the relationship two-ways.
```
Comment (steffilazerte) [inline content/blog/2023-12-22-news-december-2023/index.Rmd:618]:
```suggestion
It's a take from the maintainer perspective, on how maintainers of reverse dependencies can help make the relationship a two-way one.
```

File: content/blog/2023-12-22-news-december-2023/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,645 @@
...
+This interview was furthermore featured on the [R Weekly highlights podcast](https://rweekly.fireside.fm/145?t=848) hosted by Eric Nantz and Mike Thomas.
+
+### "A package a day" series on Mastodon!
+
+We've started sharing about one package a day from our federated community of open source software developers, on [Mastodon](https://mastodon.social/@rOpenSci@hachyderm.io/111522245481433695)!
+Follow us on Mastodon, or search for `from:@rOpenSci@hachyderm.io #apackageaday` to find all posts until now.
+
+### Resources for [multilingual publishing](/commcalls/nov2023-multilingual/) community call
+
+On this [community call](/commcalls/nov2023-multilingual/), [Maëlle Salmon](/author/ma%C3%ABlle-salmon/), [Paola Corrales](/author/paola-corrales/), and [Elio Campitelli](/author/elio-campitelli/), shared the rOpenSci Multilingual project details. Maëlle presented the R packages that allow us to have our content in several languages. Elio and Paola shared about the translation workflow and showed the Translation Guide written to document the process.
```
Comment (steffilazerte) [inline content/blog/2023-12-22-news-december-2023/index.Rmd:73]:
```suggestion
On this [community call](/commcalls/nov2023-multilingual/), [Maëlle Salmon](/author/ma%C3%ABlle-salmon/), [Paola Corrales](/author/pao-corrales/), and [Elio Campitelli](/author/elio-campitelli/), shared the rOpenSci Multilingual project details. Maëlle presented the R packages that allow us to have our content in several languages. Elio and Paola shared about the translation workflow and showed the Translation Guide written to document the process.
```

File: content/blog/2023-12-22-news-december-2023/index.md [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+
+    * [GLMMcosinor](https://github.com/ropensci/software-review/issues/603), Fit a cosinor model using a generalised mixed modelling framework. Submitted by [Rex Parsons](https://rwparsons.github.io/). 
+
+    * [octolog](https://github.com/ropensci/software-review/issues/502), Better Github Action Logging. Submitted by [Jacob Wujciak-Jens](https://github.com/assignUser). 
+
+* Nine at ['3/reviewer(s)-assigned'](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A3/reviewer(s)-assigned):
+
+     * [rgeeExtra](https://github.com/ropensci/software-review/issues/608), Extensions for rgee. Submitted by [Cesar Aybar](http://csaybar.github.io). 
+
+    * [fellingdateR](https://github.com/ropensci/software-review/issues/618), Estimate, report and combine felling dates of historical tree-ring. Submitted by [KriHa](https://wwww.researchgate.net/profile/Kristof_Haneca).
```
Comment (steffilazerte) [inline content/blog/2023-12-22-news-december-2023/index.md:131]:
```suggestion
    * [fellingdateR](https://github.com/ropensci/software-review/issues/618), Estimate, report and combine felling dates of historical tree-ring. Submitted by [KriHa](https://www.researchgate.net/profile/Kristof_Haneca). 
```

File: content/blog/2023-12-22-news-december-2023/index.md [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+
+    * [octolog](https://github.com/ropensci/software-review/issues/502), Better Github Action Logging. Submitted by [Jacob Wujciak-Jens](https://github.com/assignUser). 
+
+* Nine at ['3/reviewer(s)-assigned'](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A3/reviewer(s)-assigned):
+
+     * [rgeeExtra](https://github.com/ropensci/software-review/issues/608), Extensions for rgee. Submitted by [Cesar Aybar](http://csaybar.github.io). 
+
+    * [fellingdateR](https://github.com/ropensci/software-review/issues/618), Estimate, report and combine felling dates of historical tree-ring. Submitted by [KriHa](https://wwww.researchgate.net/profile/Kristof_Haneca). 
+
+    * [rOPTRAM](https://github.com/ropensci/software-review/issues/612), Derive soil moisture using the OPTRAM algorithm. Submitted by [Micha Silver](https://www.surfaces.co.il).
```
Comment (steffilazerte) [inline content/blog/2023-12-22-news-december-2023/index.md:133]: I get a warning about this site (surfaces.co.il) not being secure, so I can't test that the link works...

File: content/blog/2023-12-22-news-december-2023/index.md [hunk 8]
```diff
@@ -0,0 +1,214 @@
...
+
+     * [rgeeExtra](https://github.com/ropensci/software-review/issues/608), Extensions for rgee. Submitted by [Cesar Aybar](http://csaybar.github.io). 
+
+    * [fellingdateR](https://github.com/ropensci/software-review/issues/618), Estimate, report and combine felling dates of historical tree-ring. Submitted by [KriHa](https://wwww.researchgate.net/profile/Kristof_Haneca). 
+
+    * [rOPTRAM](https://github.com/ropensci/software-review/issues/612), Derive soil moisture using the OPTRAM algorithm. Submitted by [Micha Silver](https://www.surfaces.co.il). 
+
+    * [baRulho](https://github.com/ropensci/software-review/issues/609), Quantifying (Animal) Sound Degradation. Submitted by [Marcelo Araya-Salas](https://marce10.github.io/). 
+
+    * [rgeeExtra](https://github.com/ropensci/software-review/issues/608), Extensions for rgee. Submitted by [Cesar Aybar](http://csaybar.github.io).
```
Comment (steffilazerte) [inline content/blog/2023-12-22-news-december-2023/index.md:137]:
```suggestion
    * [rgeeExtra](https://github.com/ropensci/software-review/issues/608), Extensions for rgee. Submitted by [Cesar Aybar](https://csaybar.github.io). 
```

File: content/blog/2023-12-22-news-december-2023/index.md [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+
+     * [comtradr](https://github.com/ropensci/software-review/issues/613), Interface with the United Nations Comtrade API. Submitted by [paulbochtler](https://github.com/datapumpernickel). 
+
+    * [GLMMcosinor](https://github.com/ropensci/software-review/issues/603), Fit a cosinor model using a generalised mixed modelling framework. Submitted by [Rex Parsons](https://rwparsons.github.io/). 
+
+    * [octolog](https://github.com/ropensci/software-review/issues/502), Better Github Action Logging. Submitted by [Jacob Wujciak-Jens](https://github.com/assignUser). 
+
+* Nine at ['3/reviewer(s)-assigned'](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A3/reviewer(s)-assigned):
+
+     * [rgeeExtra](https://github.com/ropensci/software-review/issues/608), Extensions for rgee. Submitted by [Cesar Aybar](http://csaybar.github.io).
```
Comment (steffilazerte) [inline content/blog/2023-12-22-news-december-2023/index.md:129]:
```suggestion
     * [rgeeExtra](https://github.com/ropensci/software-review/issues/608), Extensions for rgee. Submitted by [Cesar Aybar](https://csaybar.github.io). 
```

File: content/blog/2023-12-22-news-december-2023/index.Rmd [hunk 10]
```diff
@@ -0,0 +1,645 @@
+---
+title: 'rOpenSci News Digest, December 2023'
+author: 'The rOpenSci Team'
+date: '2023-12-22'
+tags:
+  - newsletter
+slug: news-december-2023
+description: donate to rOpenSci; deposits package for data publishing; a package a day; new packags and package news
```
Comment (steffilazerte) [inline content/blog/2023-12-22-news-december-2023/index.Rmd:8]:
```suggestion
description: donate to rOpenSci; deposits package for data publishing; a package a day; new packages and package news
```

**Review by steffilazerte — COMMENTED** [review]
Got it done before I left after all! 🎉 
Looks good, just a couple of minor bits, and I love the developer corner as well. 
Thanks!
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md

**Comment by maelle** [comment]
Thanks @steffilazerte and @yabellini!

File: content/blog/2023-12-22-news-december-2023/index.Rmd [hunk 11]
```diff
@@ -0,0 +1,655 @@
...
+
+Sustaining an open project with quality infrastructure freely accessible to the global and diverse community of R software users, research software developers, and engineers requires many different resources. Our organization’s ongoing costs are supported by grants and donations from individuals and organizations which share our vision and mission.
+
+By supporting us, you’re not just donating; you’re contributing to a community that’s breaking down barriers, creating opportunities, and shaping the future of open and reproducible science for everyone. 
+
+Join us in this vital mission today! You can donate here: https://ropensci.org/donate/
+
+We are grateful for the donations we have already received. 
+
+### Easier data publishing: An interview of Mark Padgham about the deposits package!
```
Comment (maelle) [inline content/blog/2023-12-22-news-december-2023/index.Rmd:58]: @mpadge ok with this summary?

File: content/blog/2023-12-22-news-december-2023/index.Rmd (hunk 2 above)
Comment (yabellini) [reply content/blog/2023-12-22-news-december-2023/index.Rmd:66]: Link to one of the post on LinkedIn: https://www.linkedin.com/feed/update/urn:li:activity:7140023562223943680

File: content/blog/2023-12-22-news-december-2023/index.Rmd [hunk 12]
```diff
@@ -0,0 +1,655 @@
...
+
+rOpenSci Mark Padgham was interviewed on the R Consortium blog about his [deposits package](https://www.r-consortium.org/blog/2023/11/30/deposits-r-package-delivers-a-common-workflow-for-r-users) that simplifies data sharing.
+
+> "Publicly depositing datasets associated with published research is becoming more common, partly due to journals increasingly requiring data sharing and partly through more general and ongoing cultural changes with data sharing. Yet data sharing is often seen as time-consuming, particularly to meet the expectations of individual data repositories. While documentation and training can help familiarize users with data-sharing processes, browser-based data, and metadata submission workflows can only be so fast, are not easily reproduced, and do not facilitate regular or automated data and metadata updates. Better programmatic tools can transform data sharing from a mountainous climb into a pit of success."
+
+This interview was furthermore featured on the [R Weekly highlights podcast](https://rweekly.fireside.fm/145?t=848) hosted by Eric Nantz and Mike Thomas.
+
```
Comment (yabellini) [inline content/blog/2023-12-22-news-december-2023/index.Rmd:65]:
```suggestion
## rOpenSci Developer Guide Translation to Portuguese 

We want to express our gratitude to the community members who have been actively contributing to the localization and translation of the rOpenSci Developer Guide into Portuguese. Your dedication and efforts are making this valuable resource accessible to an even wider audience.  Here's a quick update on the progress:

*    18% of the book is already translated! 📚
*    2 chapters are awaiting a second review. 👀
*    24 chapters are still awaiting translation. 🌐
*    7 amazing contributors are driving this initiative: _Pedro Duarte Faria, Daniel Vartanian, Marcelo Perlin, Beatriz Milz, João Granja-Correia, António Pedro and Mariana._ 👏  Thank you for the incredible work you've done so far! 🙌

We're not done yet, and we invite more members of our community to join this collaborative effort. If you want to contribute to the translation project, join us on GitHub: [rOpenSci Developer Guide Translation Project](https://github.com/orgs/ropensci/projects/7). 🚀

```

Comment (yabellini) [reply content/blog/2023-12-22-news-december-2023/index.Rmd:65]: My intention is to have some chapter for the first review ready before we sent the newsletter. If that is the case, then some of the number can change.

**Pull request merged**